### PR TITLE
DEVDOCS-4847: [update] Building Storefront Channels, update protected UI settings

### PR DESCRIPTION
**Pull request by slsriehl**
# [DEVDOCS-4847]

## What changed?
Provide a bulleted list in the present tense
* Update the list of available protected UI sections for channel menus
* Updates for style and clarity

## Anything else?
* [api-specs PR #1319](https://github.com/bigcommerce/api-specs/pull/1319)

ping {names}

[DEVDOCS-4847]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4847?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/api-docs/channels/channels-storefront-tutorial.mdx [hunk 1]
```diff
@@ -11,7 +11,7 @@ This article documents how to use [Channels Toolkit](/api-docs/channels/guide/ch
 
 ## Creating a channel
 
-After receiving the [POST response](/api-docs/apps/guide/auth#receiving-the-post-response) at the end of the [single-click app OAuth flow](/api-docs/apps/guide/auth#receiving-the-post-response), create a channel on the merchant's store. This allows merchants to assign product listings to the storefront and configure storefront specific settings. To create a storefront channel, set `type` to `storefront` in the [create a channel](/docs/rest-management/channels#create-a-channel) request.
+After receiving the final response of the [single-click app OAuth flow](/api-docs/apps/guide/auth#receiving-the-post-response) `auth` callback, create a channel on the merchant's store. This allows merchants to assign product listings to the storefront and configure storefront specific settings. To create a storefront channel, set `type` to `storefront` in a request to the [Create a channel](/docs/rest-management/channels#create-a-channel) endpoint.
```
Comment (bc-traciporter) [inline docs/api-docs/channels/channels-storefront-tutorial.mdx:14]:
```suggestion
After receiving the final response of the [single-click app OAuth flow](/api-docs/apps/guide/auth#receiving-the-post-response) `auth` callback, create a channel on the merchant's store. This channel allows merchants to assign product listings to the storefront and configure storefront-specific settings. To create a storefront channel, set `type` to `storefront` in a request to the [Create a channel](/docs/rest-management/channels#create-a-channel) endpoint.
```

File: docs/api-docs/channels/channels-storefront-tutorial.mdx [hunk 2]
```diff
@@ -89,21 +86,23 @@ Accept: application/json
...
+|:------|:------|:------------|
+| Storefront Settings | `storefront_settings` | Renders channel-specific storefront settings. |
+| Carousel | `carousel` | Renders channel-specific carousel settings. |
+| Currencies | `currencies` | Renders channel-specific currency settings. |
+| Domains | `domains` | Renders channel-specific domain settings. |
+| Notifications | `notifications` | Renders channel-specific notification settings. |
+| Social | `social` | Renders channel-specific social media settings. |
 
-Include protected sections in the [create channel menus request](/docs/rest-management/channels/channel-menus#create-channel-menus) to display BigCommerce-provided channel-specific settings pages.
+The term _protected section_ refers to the settings in that section being protected from the settings selected at the storewide level. Include these protected sections in requests to the [Create channel menus](/docs/rest-management/channels/channel-menus#create-channel-menus) endpoint to display BigCommerce-provided, channel-specific settings pages and corresponding menu items. Enabled protected sections precede custom sections in a channel's menu.
```
Comment (bc-traciporter) [inline docs/api-docs/channels/channels-storefront-tutorial.mdx:105]:
```suggestion
The term _protected section_ refers to secure areas that are protected from storewide-level selections. Include these protected sections in requests to the [Create channel menus](/docs/rest-management/channels/channel-menus#create-channel-menus) endpoint to display BigCommerce-provided, channel-specific settings pages and corresponding menu items. Enabled protected sections precede custom sections in a channel's menu.
```

File: docs/api-docs/channels/channels-storefront-tutorial.mdx [hunk 3]
```diff
@@ -121,17 +120,15 @@ Accept: application/json
...
+Include the `storefront_settings` [protected section](#protected-ui-sections) in a Channel Menu's `bigcommerce_protected_app_sections` array to render the BigCommerce-provided **Storefront Settings** menu item in the channel's navigation and corresponding settings page, and protect channel storefront settings from those set for the store in general.
 
 ![Channel Storefront Settings](https://storage.googleapis.com/bigcommerce-production-dev-center/images/channels/channels-sf-storefront-settings.png "Channel Storefront Settings")
 
-Read channel specific storefront settings using the [Settings API](/docs/rest-management/settings). For example, to [get storefront category settings](/docs/rest-management/settings/storefront-category#get-storefront-category-settings), send a `GET` request to `/v3/settings/storefront/category`.
+Read channel specific storefront settings using the [Settings API](/docs/rest-management/settings). For example, you can see a storefront's category settings using the [Get storefront category settings](/docs/rest-management/settings/storefront-category#get-storefront-category-settings) endpoint.
```
Comment (bc-traciporter) [inline docs/api-docs/channels/channels-storefront-tutorial.mdx:131]:
```suggestion
Read channel-specific storefront settings using the [Settings API](/docs/rest-management/settings). For example, you can see a storefront's category settings using the [Get storefront category settings](/docs/rest-management/settings/storefront-category#get-storefront-category-settings) endpoint.
```

**Review by bc-traciporter — DISMISSED** [review]
I made some editorial suggestions. Feel free to take them or leave them.

File: docs/api-docs/channels/channels-storefront-tutorial.mdx [hunk 4]
```diff
@@ -89,21 +86,23 @@ Accept: application/json
 ```
 
 <Callout type="info">
-  These UI sections were previously managed via `config_meta`. As of March 29, 2022, changes to UI sections via `config_meta` are dual written to the new API when `config_meta.app.sections` exists in the payload.
+  These UI sections were previously managed using `config_meta`. As of March 29, 2022, changes to UI sections that use `config_meta` are dual-written to the Channel Menus object when `config_meta.app.sections` exists in the payload.
```
Comment (slsriehl) [inline docs/api-docs/channels/channels-storefront-tutorial.mdx:89]:
```suggestion
  Previously, you could manage these settings using the `config_meta` property on the Channels object. As of March 29, 2022, changes to UI sections for channels that already have a non-null `config_meta.app.sections` property are dual-written to the Channel Menus object.
```

[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/api-docs/channels/channels-storefront-tutorial.mdx:89]: will remove the passive voice.

File: docs/api-docs/channels/channels-storefront-tutorial.mdx (hunk 2 above)
Comment (slsriehl) [reply docs/api-docs/channels/channels-storefront-tutorial.mdx:105]: this is a bit word salady, isn't it. thanks, will revise.

Comment (slsriehl) [inline docs/api-docs/channels/channels-storefront-tutorial.mdx:105]:
```suggestion
The term _protected section_ refers to the settings on the corresponding menu pages. Settings in protected sections override the default settings configured at the storewide level. Include these protected sections in requests to the [Create channel menus](/docs/rest-management/channels/channel-menus#create-channel-menus) endpoint to display BigCommerce-provided, channel-specific settings pages and corresponding menu items. Enabled protected sections precede custom sections in a channel's menu.
```

File: docs/api-docs/channels/channels-storefront-tutorial.mdx (hunk 1 above)
Comment (slsriehl) [inline docs/api-docs/channels/channels-storefront-tutorial.mdx:14]:
```suggestion
After receiving the final response of the [single-click app OAuth flow](/api-docs/apps/guide/auth#receiving-the-post-response) `auth` callback, create a channel on the merchant's store. Creating a dedicated channel for a storefront allows merchants to assign the storefront selected product listings and configure storefront-specific settings. To create a storefront channel, set `type` to `storefront` in a request to the [Create a channel](/docs/rest-management/channels#create-a-channel) endpoint.
```

**Pull request merged**